src/cli_plugin/install/__tests__/download.js

```diff
@@ -0,0 +1,226 @@
+import expect from 'expect.js';
+import sinon from 'sinon';
+import nock from 'nock';
+import glob from 'glob-all';
+import rimraf from 'rimraf';
+import mkdirp from 'mkdirp';
+import Logger from '../../lib/logger';
+import { download, _downloadSingle } from '../download';
+import { join } from 'path';
+
+describe('kibana cli', function () {
+
+  describe('plugin downloader', function () {
+    const testWorkingPath = join(__dirname, '.test.data');
```

Comment:
If you do end up keeping the `mkdirp`/current directory approach, please add this path to the `.gitignore` for when the tests are killed.